scripts/visualization/opf_visualizer.js

```diff
@@ -378,13 +392,15 @@ angular.module('app').controller('AppCtrl', ['$scope', '$timeout', function($sco
           }
           $scope.$apply();
         },
+        // drawing the graph itself
         drawCallback: function(graph, is_initial) {
           if (is_initial) {
             setColors(graph.getColors());
           }
-        }
-      }
-    );
+        },
+        // highlighting
+        underlayCallback: highlightPoint(canvas, area, graph),
```

Comment:
@jefffohl I'm stuck here on the scopes/names. Could you help me to access the respective variables in our code? 
The example is from http://dygraphs.com/gallery/#g/highlighted-weekends